ajax/libs/printThis/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "printThis",
+  "description": "Printing plug-in for jQuery",
+  "filename": "printThis.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jasonday/printThis.git"
+  },
+  "keywords": [
+    "print",
+    "jQuery"
```

Comment:
I think you can add some keywords for the author to help users find the library more easily :+1: